packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/utils.dart

```diff
@@ -23,12 +24,19 @@ double min(Iterable<double> numbers) =>
 double max(Iterable<double> numbers) =>
     numbers.fold(-double.infinity, (minimum, cur) => math.max(minimum, cur));
 
-String crossAxisAssetImageUrl(CrossAxisAlignment alignment) {
-  return 'assets/img/story_of_layout/cross_axis_alignment/${describeEnum(alignment)}.png';
+String flexType(Axis direction) {
+  if (direction == Axis.horizontal) return 'Row';
```

Comment:
nit: change to
`=> direction == Axis.horizontal ? 'Row' : 'Column';`
and double check we don't have another helper returning 'Row' and 'Column'.
This could even be an extension method on the Axis enum.

Reply:
great catch and great suggestion, didn't know I can write extension on enum as well!